pkgs/applications/misc/plots/default.nix

```diff
@@ -0,0 +1,77 @@
+{ fetchFromGitHub
+, gobject-introspection
+, lib
+, libadwaita
+, python3Packages
+, python3
```

Comment:
I'm not sure if there's a style guide/recommendation about this, but given that `python3Packages.python == python3` and `python3.pkgs == python3Packages`, it looks slightly odd to me to take both as arguments when only one of them is needed. (Without any more data I'm not saying you need to change it, but just wanted to point it out.)